go/service/teams_test.go

```diff
@@ -0,0 +1,24 @@
+package service
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseSeitanToken(t *testing.T) {
+	token := "aazaaa0a+aaaaaaaa"
+	s := fmt.Sprintf("HELLO AND WELCOME TO THIS TEAM. token: %s", token)
+	require.Equal(t, token, parseInviteToken(s))
+	token = "aazaaa0aaaaaaaaa"
+	s = fmt.Sprintf("HELLO AND WELCOME TO THIS TEAM. token: %s", token)
+	require.Equal(t, token, parseInviteToken(s))
+	s = token
+	require.Equal(t, token, parseInviteToken(s))
+	s = fmt.Sprintf("%s %s", token, token)
+	require.NotEqual(t, token, parseInviteToken(s))
+	s = "token: MIKE"
+	require.NotEqual(t, "MIKE", parseInviteToken(s))
+	require.Equal(t, s, parseInviteToken(s))
```

Comment:
I know this will pass with your regexp, but something along the lines of:

    invited to team 0123456789012345 with token: %s

just to make sure that team name never passes as a token.

Reply:
Added another check, let me know if it is not what you had in mind.

Reply:
LGTM